src/Analyzers/CSharp/CodeFixes/MisplacedUsingDirectives/MisplacedUsingDirectivesCodeFixProvider.cs

```diff
@@ -186,8 +186,8 @@ void Recurse(SyntaxList<MemberDeclarationSyntax> members)
 
         private static async Task<SyntaxNode> ExpandUsingDirectiveAsync(Document document, UsingDirectiveSyntax usingDirective, CancellationToken cancellationToken)
         {
-            var newType = await Simplifier.ExpandAsync(usingDirective.Type, document, cancellationToken: cancellationToken).ConfigureAwait(false);
-            return usingDirective.WithType(newType);
+            var newType = await Simplifier.ExpandAsync(usingDirective.NamespaceOrType, document, cancellationToken: cancellationToken).ConfigureAwait(false);
+            return usingDirective.WithNamespaceOrType(newType);
```

Comment:
fallout of API review meeting.  We decided to rename 'Type' to 'NamespaceOrType' (to match INamespaceOrTypeSymbol)